core/sr-arithmetic/src/parts_per_x.rs

```diff
@@ -0,0 +1,551 @@
+
```

Comment:
header missing + how do you like the name `per_things.rs`? we used to call them like this, even in the code.

Reply:
Please add the licence header. 🏴